### ISSUE TITLE
cannot export custom yolov5 .pt model

### ISSUE THREAD

**Issue by HoangTienDuc** [zhiqwang/yolov5-rt-stack#6]
Hello @zhiqwang . Thank for awesome work.
i am successful with exporting yolov5s_pretrained.pt to onnx model. But i cannot export my custom yolov5l.pt model.
How can you export your custom model?

**Comment by zhiqwang** [comment]
Hi @HoangTienDuc , Also thanks for your sharing

I didn't working on ultralytics's `yolov5l` now, hope to fix `yolov5m` and `yolov5l` this week.

**Comment by zhiqwang** [comment]
Hi @HoangTienDuc , The latest released [`v0.2.4`](https://github.com/zhiqwang/yolov5-rt-stack/releases/tag/v0.2.4) support loading `yolov5m.yaml` and `yolov5l.yaml`. The new interface to load `yolov5l` is as follows:

```python
from hubconf import yolov5
model = yolov5(cfg_path='yolov5l.yaml', pretrained=True)
model.eval()
x = [torch.rand(3, 416, 320), torch.rand(3, 480, 352)]
predictions = model(x)
```

And a important thing to be noted here, if you train your model using ultralytics's repo, you shoud update the model checkpoint to do inference in `yolov5-rt-stack`, and ultralytics's trained model has a limitations that their model must load in the root path of ultralytics, so another important thing is to desensitize the path befor updating ultralytics's trained model as follows:

```python
checkpoints_ = torch.load(weights, map_location='cpu')['model']
torch.save(checkpoints_.state_dict(), './checkpoints/yolov5/yolov5s_ultralytics.pt')
```

and then convert ultralytics's trained (or your own) model checkpoint with the following scripts:

```bash
python -m utils.updated_checkpoint [--checkpoint_path ./yolov5s.pt]
                                   [--cfg_path ./models/yolov5s.yaml]
```
                                   [--updated_checkpoint_path ./checkpoints/yolov5/yolov5s.pt]

**Comment by zhiqwang** [comment]
I made some changes, now the model can be loaded as below:

```python
from hubconf import yolov5l
model = yolov5l(pretrained=True)
model.eval()
x = [torch.rand(3, 416, 320), torch.rand(3, 480, 352)]
predictions = model(x)
```

or simply use `torch.hub`:

```python
model = torch.hub.load('zhiqwang/yolov5-rt-stack', 'yolov5l', pretrained=True)
model.eval()
x = [torch.rand(3, 416, 320), torch.rand(3, 480, 352)]
predictions = model(x)
```
